### PR TITLE
fix: #239 make sure parent is proband again when removed from group

### PR DESCRIPTION
**Pull request by evans-g-crsj**
# [BUG] [Make sure parent is proband again when removed from family]

closes #[239]

## Description
when a parent is removed from a family (group), it returns to a new family (consisting only of him/her as members). Him or Her must be proband.

Critères de succès
a member is proband when removed from a family
## Validation

- [ ] Test Coverage
- [ ] QA Done
- [ ] Design/UI Approved from design

## Screenshot
before
image
after
image

## QA
remove a parent and make sure the parent becomes a probant

### CODE REVIEW

File: client/src/helpers/fhir/patientHelper.ts [hunk 1]
```diff
@@ -15,16 +18,20 @@ export function formatRamq(value: string): string {
   return newValue
     .replace(/\s/g, '')
     .split('')
-    .reduce((acc, char, index) => ((char !== ' ' && [3, 7]
-      .includes(index)) ? `${acc}${char} ` : `${acc}${char}`), '').trimEnd();
+    .reduce(
```
Comment (evans-g-crsj) [inline client/src/helpers/fhir/patientHelper.ts:21]: eslint

File: client/src/helpers/fhir/patientHelper.ts [hunk 2]
```diff
@@ -15,16 +18,20 @@ export function formatRamq(value: string): string {
...
+    .trimEnd();
 }
 
 export type RamqDetails = {
-  startFirstname?: string
-  startLastname?: string
-  sex?: 'male' | 'female'
-  birthDate?: Date
-}
+  startFirstname?: string;
```
Comment (evans-g-crsj) [inline client/src/helpers/fhir/patientHelper.ts:30]: eslint

File: client/src/helpers/fhir/patientHelper.ts [hunk 3]
```diff
@@ -4,7 +4,10 @@ import moment from 'moment';
 import { isValidRamq } from './api/PatientChecker';
```
Comment (evans-g-crsj) [inline client/src/helpers/fhir/patientHelper.ts:4]: a lot of eslint auto formatting

**Review by claudia1296 — APPROVED** [review]

**Review by francisl — APPROVED** [review]
🍺

**Pull request merged**